### DISCUSSION TITLE
will it support for version 3.0 ?

### DISCUSSION THREAD

**Discussion by mohandtharsha** [Jacobtims/filament-breezy#286] [Q&A]
will it support for version 3.0 because I am getting the error while installing this package.

Below is the error,

> @php artisan package:discover --ansi

   BadMethodCallException

  Method Filament\Panel::getDomains does not exist.

  at vendor\filament\support\src\Concerns\Macroable.php:72
     68▕     {
     69▕         $macro = static::getMacro($method);
     70▕
     71▕         if ($macro === null) {
  ➜  72▕             throw new BadMethodCallException(sprintf(
     73▕                 'Method %s::%s does not exist.',
     74▕                 static::class,
     75▕                 $method,
     76▕             ));

  i   Bad Method Call: Did you mean Filament\Panel::getDomain() ?

  1   vendor\jeffgreco13\filament-breezy\routes\web.php:10
      Filament\Support\Components\Component::__call("getDomains", [])

  2   vendor\laravel\framework\src\Illuminate\Routing\Router.php:502
      Illuminate\Support\ServiceProvider::{closure}(Object(Illuminate\Routing\Router))

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1

**Comment by jeffgreco13** [comment]
Please upgrade to the latest version of Breezy: `composer update`
